exporters/ostream/test/ostream_metric_test.cc

```diff
@@ -151,6 +156,11 @@ TEST(OStreamMetricsExporter, ExportHistogramPointData)
       "\n  counts     : [200, 300, 400, 500, ]"
       "\n  attributes\t\t: "
       "\n\ta1: b1"
+      "\n  resources\t:"
+      "\n\tservice.name: unknown_service"
+      "\n\ttelemetry.sdk.version: 1.5.0"
```

Comment:
Can we use the macro `OPENTELEMETRY_SDK_VERSION` instead of the hardcoding version? This way, we don't need to modify this part of the code with every new release.

Reply:
Good call. Done. There are some strange cmake failures that I'm having difficulties reproducing on my ubuntu 20.04 machine locally, but they are also not marked as "required" on the CI stages...

Reply:
@lalitb: What are the next steps I need to take here in order to debug? It's not clear to me what the next debugging steps are (or how some of these CI failures are relevant to a largely NFC patch like this one).

Reply:
@lalitb: What are the next steps I need to take here in order to debug? It's not clear to me what the next debugging steps are (or how some of these CI failures are relevant to a largely NFC patch like this one).